openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -603,6 +625,141 @@ Location: https://client.example.net/cb?
+       |<----------------------------------------------------------------------------------|  (4) (redirect_uri)
+       |                              |                        |                           |    
+       |----------------------------->|  (4) Authorization     |                           |
+       |                              |      Challenge         |                           |
+       |                              |      Request           |                           |
+       |                              |      (auth_session,    |                           |
+       |                              |      redirect_uri)     |                           |
+       |                              |                        |                           |    
+       |<-----------------------------|  (5) Authorization     |                           |
+       |                              |      Challenge         |                           |
+       |                              |      Response (code)   |                           |
+       |                              |                        |                           |
+       |----------------------------->|  (6) Token Request     |                           |
+       |                              |      (code)            |                           |
+       |                              |                        |                           |
+       |<-----------------------------|  (7) Token Response    |                           |
+       |                              |      (Access Token)    |                           |
+       |                              |                        |                           |
+       |                              |                        |                           |
+       |  (8) Credential Request      |                        |                           |
+       |      (Access Token, proof(s))|                        |                           |
+       |------------------------------------------------------>|                           |
+       |                              |                        |                           |
+       |  (9) Credential Response     |                        |                           |
+       |      with Credential(s) OR   |                        |                           |
+       |      Transaction ID          |                        |                           |
+       |<------------------------------------------------------|                           |
+~~~
+!---
+Figure: Issuance using Authorization Challenge Endpoint
+
+
+
+## Authorization Challenge Request {#authorization-challenge-request}
+
+The request to the Authorization Challenge Endpoint is formed and sent in the same way as PAR request as defined in [@!RFC9126.html, Section 2.1], with the following additions:
+
+ - In case a wallet attestation is required by the Issuer, it has to be included in this request.
```

Comment:
Given that the wallet attestation is a form of client authentication, I think this is actually no different to the PAR endpoint which also requires client authentication?

Reply:
Fair point.